source/themes/simple/SimpleSelect.scss

```diff
@@ -0,0 +1,81 @@
+@import "config";
+@import "mixins/arrow";
+
+.select {
+  position: relative;
+}
+
+.selectInput {
+  cursor: pointer;
+
+  &:after {
+    content: "";
+    position: absolute;
+    bottom: 20px;
+    left: calc(100% - 32px);
+    background: url('./assets/select-down-arrow.svg') no-repeat;
+    width: 11px;
+    height: 7px;
+    -webkit-background-size: 11px;
+    background-size: 11px;
+  }
+}
+
+.options {
+  position: relative;
+  font-size: 16px;
+  font-family: $theme-font-light;
+  border: 1px solid #c6cdd6;
+  list-style: none;
+  padding: 0;
+  border-radius: 2px;
```

Comment:
@DominikGuzei I would add `box-sizing: border-box;` here as currently options dropdown is 2px wider than the select field (due to width:100% + border combo).